.github/workflows/cypress.yml

```diff
@@ -0,0 +1,49 @@
+name: Cypress Tests
+on:
+  workflow_dispatch:
+    inputs:
+      browser:
+        description: Cypress browser to use
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    if: github.event.deployment_status.state == 'success' || github.event_name == 'workflow_dispatch'
+    steps:
+      - name: Get PR Number
+        uses: actions/github-script@v6
+        id: pr-number
+        with:
+          result-encoding: json
+          script: |
+            try {
+              const results = await github.rest.search.issuesAndPullRequests({
+                q: context.sha
+              });
+              const prs = [...results.data.items].sort((a, b) => a.score - b.score);
+              if (prs[0]) {
+                return prs[0].number;
+              }
+            } catch (e) {
+              console.error(e);
+            }
+
```

Comment:
I thought about it, and I'm going to remove the workflow file for now, because it's technically also broken even for workflow_dispatch because the install step fails. We can re-introduce once the other issues are resolved, I'll create a separate ticket to add back in later.